source/blog.html.haml

```diff
@@ -0,0 +1,32 @@
+---
+title: Blog
+pageable: true
+per_page: 5
+---
+
+%h2.blog_heading
+  Euro Team Outreach Blog Articles
+
+%section.narrow_container
+  - page_articles.each_with_index do |article, i|
+    - pretty_date = article.date.strftime("%b %d, %Y")
+
+    %article.article-preview
+      .article-preview__title
+        %h3
+          = link_to article.title, article
+
+      .article-preview__meta
+        - if article.data.author?
+          .meta-author
+            = social_link_for(article.data.author)
+
+        %time.meta-date{ datetime: "#{article.date}" }= pretty_date
+
+      .article-preview__content
+        = article.summary
+        = link_to "Read more", article
+
+  - if paginate && num_pages > 1
```

Comment:
This shouldn't be here until we have the partial also. We should probably handle pagination in a separate PR.